src/components/EditableFields/AutoSave.tsx

```diff
@@ -104,3 +96,5 @@ export const AutoSave = ({
     </>
   )
 }
+
+export default AutoSave
```

Comment:
Is export default the preferred way of exporting a function/component? I liked that export const would ensure we use the same name at import.

Reply:
I prefer exporting the component. Maybe an ADR conversion or best practice? Decide as a team how we manage exporting React Components. There's nothing to stop using the export const function and having the default as well.